cli/lib/nf3.mjs

```diff
@@ -1414,7 +1416,7 @@ class Nf3 {
       address,
     });
     const txDataToSign = res.data;
-    return this.submitTransaction(txDataToSign);
+    return this.submitTransaction(txDataToSign, this.kycContractAddress);
```

Comment:
groupId never used

Reply:
Good spot, thanks! `X509.sol` should give itself a groupId.

Reply:
I've removed this, and related, whitelist functions entirely because they are no longer required - `X509.sol` subsumes this functionality.